test/e2e/framework/sessionconfig.go

```diff
@@ -103,6 +120,10 @@ func (cfg SessionConfig) forwardFAR(farID uint32) *ie.IE {
 	if cfg.ForwardingPolicyID != "" {
 		fwParams = append(fwParams, ie.NewForwardingPolicy(cfg.ForwardingPolicyID))
 	}
+	if cfg.NatPoolName != "" {
+		fwParams = append(fwParams, newVendorSpecificU8IE(0x8000|BBF_TYPE_APPLY_ACTION, BBF_EID, BBF_APPLY_ACTION_NAT))
+		fwParams = append(fwParams, newVendorSpecificStringIE(0x8000|BBF_TYPE_NAT_PORT_BLOCK, BBF_EID, cfg.NatPoolName))
```

Comment:
Please don't use magic numbers w/o comments. Better add constants instead of `0x8000`

Reply:
Added to constants with an explanation